### PR TITLE
Subscription export

### PR DESCRIPTION
**Pull request by tank157**
Last bit to add to the export

### CODE REVIEW

**Comment by tank157** [comment]
Assigning to @danielglauser

File: test/election_mail_works/handlers/subscription_test.clj [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+(deftest delete-handler*-test
+  (testing "deletes a subscription"
+    (let [user-id (random-uuid)
+          create (create-handler* (async/chan) {:user-id user-id})]
+      (is (= :ok (:status create)))
+      (let [event-chan (async/chan)
+            delete (delete-handler* event-chan {:user-id user-id})]
+        (is (= :ok (:status delete)))
+        (is (= {:user-id user-id} (async/<!! event-chan))))))
+  (testing "it'll delete any old thing, maybe we fix this?"
```
Comment (danielglauser) [inline test/election_mail_works/handlers/subscription_test.clj]: I'm glad you call this out in the testing comment but isn't this behavior wrong? Should we really reply with an `:ok` status when a caller tries to delete something that doesn't exist? I understand it's an unlikely case so I'd be happy with a card describing the problem.

Comment (tank157) [inline test/election_mail_works/handlers/subscription_test.clj]: Opened a new bug card to note this possible deficiency: https://www.pivotaltracker.com/story/show/144768285

**Comment by cap10morgan** [comment]
@tank157 Do we still need this PR?

**Comment by tank157** [comment]
@cap10morgan There wasn't ever a pivotal card for it, I was just trying to get the export to include the last bits of user data, but clearly this fell through the cracks. I'll see if I can tie this up on a Friday.

**Comment by tank157** [comment]
I'm going go close this down. There's been enough drift that it seems like maybe it'll be easier to re-implement this later than merge.

**Pull request closed without merging**